Source: Upstream
Upstream: Yes, r965028
Reason: See below

From c26bb9f277c733ce5b0035791ee294c923ddf06e Mon Sep 17 00:00:00 2001
From: tmcguire <tmcguire@283d02a7-25f6-0310-bc7c-ecb5cbfe19da>
Date: Thu, 7 May 2009 20:13:07 +0000
Subject: [PATCH] Backport r965026 by tmcguire from trunk to the 4.2 branch:

Merged revisions 965019 via svnmerge from
svn+ssh://tmcguire@svn.kde.org/home/kde/branches/kdepim/enterprise4/kdepim

................
  r965019 | tmcguire | 2009-05-07 21:59:01 +0200 (Thu, 07 May 2009) | 11 lines

  Merged revisions 965008 via svnmerge from
  svn+ssh://tmcguire@svn.kde.org/home/kde/branches/kdepim/enterprise/kdepim

  ........
    r965008 | tmcguire | 2009-05-07 21:50:39 +0200 (Thu, 07 May 2009) | 4 lines

    Allow the user to reply to mails in the sent-mail folder, as often requested.
    kolab/issue2150
    CCBUG: 190105
  ........
................



git-svn-id: svn://anonsvn.kde.org/home/kde/branches/KDE/4.2/kdepim@965028 283d02a7-25f6-0310-bc7c-ecb5cbfe19da
---
 kmail/kmmainwidget.cpp           |    5 +----
 kmail/messageactions.cpp         |    3 +--
 kmail/messagelistview/widget.cpp |    3 +--
 3 files changed, 3 insertions(+), 8 deletions(-)

diff --git a/kmail/kmmainwidget.cpp b/kmail/kmmainwidget.cpp
index 06ce409..79ea2a6 100644
--- a/kmail/kmmainwidget.cpp
+++ b/kmail/kmmainwidget.cpp
@@ -3450,10 +3450,7 @@ void KMMainWidget::slotMsgPopup( KMMessage &msg, const KUrl &aUrl, const QPoint
     if ( mFolder->isTemplates() ) {
       menu->addAction( mUseAction );
     } else {
-
-      if ( !mFolder->isSent() ) {
-        menu->addAction( mMsgActions->replyMenu() );
-      }
+      menu->addAction( mMsgActions->replyMenu() );
       menu->addAction( mForwardActionMenu );
     }
     menu->addAction(editAction());
diff --git a/kmail/messageactions.cpp b/kmail/messageactions.cpp
index e5bb481..869b587 100644
--- a/kmail/messageactions.cpp
+++ b/kmail/messageactions.cpp
@@ -170,8 +170,7 @@ void MessageActions::updateActions()
 {
   bool singleMsg = (mCurrentMessage != 0);
   if ( mCurrentMessage && mCurrentMessage->parent() ) {
-    if ( mCurrentMessage->parent()->isSent() ||
-         mCurrentMessage->parent()->isTemplates())
+    if ( mCurrentMessage->parent()->isTemplates() )
       singleMsg = false;
   }
   const bool multiVisible = mVisibleSernums.count() > 0 || mCurrentMessage;
diff --git a/kmail/messagelistview/widget.cpp b/kmail/messagelistview/widget.cpp
index c43f9e7..80928a6 100644
--- a/kmail/messagelistview/widget.cpp
+++ b/kmail/messagelistview/widget.cpp
@@ -618,8 +618,7 @@ void Widget::viewMessageListContextPopupRequest( const QList< Core::MessageItem
     menu.addAction( mMainWidget->useAction() );
   } else {
     // show most used actions
-    if( !folder()->isSent() )
-      menu.addAction( mMainWidget->messageActions()->replyMenu() );
+    menu.addAction( mMainWidget->messageActions()->replyMenu() );
     menu.addAction( mMainWidget->forwardMenu() );
     if( mMainWidget->sendAgainAction()->isEnabled() )
       menu.addAction( mMainWidget->sendAgainAction() );
-- 
1.6.3.1

